src/opf/ed.jl

```diff
@@ -171,6 +174,11 @@ function update!(opf::OPFModel{EconomicDispatch}, data::Dict{String,Any})
         opf.model.ext[:ptdf_iterations] = 0
         
         E = length(ref[:branch])
+        
+        tracked = findall(opf.model.ext[:tracked_branches])
+        JuMP.delete.(opf.model, opf.model[:ptdf_flow][tracked])
```

Comment:
I believe it would be sufficient to write
```suggestion
        tracked = opf.model.ext[:tracked_branches]
        JuMP.delete.(opf.model, opf.model[:ptdf_flow][tracked])
```

`tracked` will be a Vector of booleans, and you can use those to index a `Vector` with same length.

Reply:
⚠ you need to move the line `opf.model.ext[:tracked_branches] .= false` to _after_ you delete the PTDF constraints, otherwise you won't delete anything